pages/Host/HostVanDetail.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useParams, Link, NavLink, Outlet } from "react-router-dom";
+
+const HostVanDetail = () => {
+  const { id } = useParams();
+  const [currentVan, setCurrentVan] = React.useState(null);
+
+  React.useEffect(() => {
+    fetch(`/api/host/vans/${id}`)
+      .then((res) => res.json())
+      .then((data) => setCurrentVan(data.vans));
```

Comment:
You can add a way to catch errors, because if an error occurs the app will not do anything.

Reply:
@hirwaaldo1, I resolved this issue of handling the errors in chapter 4 as the scope of this doesn't cover it, so I feel Like I will be doing the double work! what do you think?